boards/xtensa/esp32/esp32-devkitc/src/esp32_bringup.c

```diff
@@ -162,91 +166,91 @@ int esp32_bringup(void)
 #ifdef CONFIG_ESP32_AES_ACCELERATOR
   ret = esp32_aes_init();
   if (ret < 0)
-    {
-      syslog(LOG_ERR,
-             "ERROR: Failed to initialize AES: %d\n", ret);
-    }
+  {
```

Comment:
This style changes are not compliant to the NuttX coding style.

Reply:
To check coding style, there is the program under tools/nxstyle.c. Just compile with:

```
cd tools
gcc nxstyle.c -o nxstyle
cd ..
```

and run it on the file in question, e.g.,

```
$ tools/nxstyle boards/xtensa/esp32/esp32-devkitc/src/esp32_bringup.c
```

It will print nothing if all is well or tell you which areas fail the check.

Reply:
@hartmannathan 
I've just learned about the nuttx/tools/checkpatch.sh and it seems to be using nxstyle in a batch manner. Can I use checkpath.sh instead of nxstyle to every-single-file?